_partials/rightcol.html.haml

```diff
@@ -2,27 +2,27 @@
   %h2 Getting Started
   %ul.nav.nav-pills.nav-stacked
     %li
-      %a{:href => href="https://www.jboss.org:80/narayana/downloads/latest.html"}
+      %a{:href => href="#{site.base_url}/downloads/index.html"}
         %i.icon-fixed-width.icon-arrow-right
         Download the latest version of Narayana standalone
     %li
-      %a{:href => href="https://ci.jboss.org/hudson/job/WildFly-latest-master/"}
+      %a{:href => href="https://www.wildfly.org/downloads/"}
         %i.icon-fixed-width.icon-arrow-right
         Or download the latest version of Narayana + WildFly
```

Comment:
@ochaloup I wonder if we should reword this a bit. "Or download the latest release of WildFly" - it may be that the version of Narayana in that release is not the latest version of Narayana - WDYT? /cc @mmusgrov

Reply:
Agreed, the narayana version in WildFly master may lag behind our latest release so your text covers this case.

By the way, the wildfly download link (https://ci.jboss.org/hudson/job/WildFly-latest-master/) does not work.

Reply:
@tomjenkinson I changed the text as you recommended.
@mmusgrov +1

The PR was updated.
I verified once again now if the links on the right column work with my changes an all seems fine.